src/scss/_themes.scss

```diff
@@ -63,7 +66,7 @@
 				padding: 0 10px;
 			};
 
-			&:nth-child(4) {
+			&:last-child {
```

Comment:
True, then it should be commented. Is there a case where this will go onto 2 rows? Or is it always going to be a single row. If the latter, `last-child` is fine